src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -355,6 +353,43 @@ def _store_testcase_for_regression_testing(testcase, testcase_file_path):
                    regression_testcase_url)
 
 
+# If this approach of using the protos directly is adopted, these two function
+# would no longer be needed and we'd make setup.preprocess_setup_testcase
+# return a proto directly.
+def setup_input_to_proto(setup_input):
+  """A temporary workaround to limit the changes inside this file."""
+  setup_input_proto = uworker_msg_pb2.SetupInput()
```

Comment:
Can we even just reach inside `SetupInput` and just return `setup_input._proto`?

Reply:
I think you are right, I should try simply that!